Project1/Index.html

```diff
@@ -0,0 +1,92 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <link rel="stylesheet" href="styles.css">
+  <title>GD2 Project 1</title>
+
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto">
+  <style>
+    body {
+      font-family: 'Roboto', sans-serif
+    }
+  </style>
+</head>
+
+
+
+<header>
+  <figure>
+    <h1>Yes, the Octopus Is Smart as Heck. But Why?</h1>
+    <h2>It has eight arms, three hearts — and a plan. Scientists aren’t sure how the cephalopods got to be so
+      intelligent.</h2>
+    <img src="images/octopus.jpg" alt="A picture of an octopus">
+    <figcaption>An octopus in an aquarium in Brest, France. The animals are highly intelligent, but researchers are
+      uncertain how the trait evolved. Fred Tanneau/Agence France-Presse — Getty Images</figcaption>
+  </figure>
+
+
+  <h3>By Carl Zimmer</h3>
+
+  <h4>Nov. 30, 2018</h4>
+</header>
+
+
+<body>
+  <p>To demonstrate how smart an octopus can be, Piero Amodio points to a <span>YouTube video</span>. It shows an
+    octopus
+    pulling two halves of a coconut shell together to hide inside. Later the animal stacks the shells together like
+    nesting bowls — and carts them away.</p>
+
+  <p>“It suggests the octopus is carrying these tools around because it has some understanding they may be useful
+    in the future,” said Mr. Amodio, a graduate student studying animal intelligence at the University of Cambridge
+    in Britain.</p>
+  <aside clearfix>
+    <blockquote>
+      "Intelligence comprises sophisticated cognitive skills that help an animal thrive."
+    </blockquote>
+  </aside>
+
+  <p>But his amazement is mixed with puzzlement.</p>
+
+  <p>For decades, researchers have studied how certain animals evolved to be intelligent, among them apes,
+    elephants, dolphins and even some birds, such as crows and parrots.</p>
+
+  <p>But all the scientific theories fail when it comes to cephalopods, a group that includes octopuses, squid and
+    cuttlefish. Despite feats of creativity, they lack some hallmarks of intelligence seen in other species.</p>
+
+  <p>“It’s an apparent paradox that’s been largely overlooked in the past,” said Mr. Amodio. He and five other
+    experts on animal intelligence explore this paradox in a paper published this month in the journal Trends in
+    Ecology and Evolution.</p>
+
+  <p>For scientists who study animal behavior, intelligence is not about acing a calculus test or taking a car
+    apart and putting it back together. Intelligence comprises sophisticated cognitive skills that help an animal
+    thrive.</p>
+
+  <figure class="body-photo cleafix">
```

Comment:
Note typo in classname: 
```suggestion
  <figure class="body-photo clearfix">
```

Reply:
See note in the CSS about this, too; you can actually delete the `clearfix` classname here (leaving `body-photo` in place, though!).